### PR TITLE
 #166789969 Bookmark Articles for Reading Later

### PR DESCRIPTION
**Pull request by IsaiahRn**
#### What does this PR do?

- Bookmark articles for reading later

#### Description of Task to be completed

- Ensure user to bookmark articles

- Add tests

#### How should this be manually tested?

1. Run the following commands in the terminal
```
git clone https://github.com/andela/ah-92explorers-frontend.git
cd ah-92explorers-frontend
git checkout ft-bookmark-articles-166789969
```

- npm install

- npm run dev

2. Browser to: http://localhost:8080/article/{slug}

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?
[#166789969](https://www.pivotaltracker.com/story/show/166789969)
#### Screenshots (if appropriate)

- N/A

#### Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

**Review by niyoceles — DISMISSED** [review]
This PR can be merged, It is doing what expected to do.

**Review by niyoceles — DISMISSED** [review]

**Review by kodek-sleuth — DISMISSED** [review]
I failed to bookmark articles on the homepage, I think as a User I must be able to bookmark articles on the feed page.

**Comment by IsaiahRn** [comment]
> I failed to bookmark articles on the homepage, I think as a User I must be able to bookmark articles on the feed page.

I am going to work on that it!

**Review by elemanhillary-zz — DISMISSED** [review]
as a user am unable to bookmark more than 1 article, kindly fix the bug causing it.

File: src/components/Articles/Feed.jsx [hunk 1]
```diff
@@ -92,8 +117,8 @@ export class Feed extends Component {
               <small style={{ color: '#ffffff' }} className="fRST">{ articles[0].date }</small>
               <div className="profileDisplay pDFRF">
                 <img src={articles[0].profile} alt="" className="profImg pIFRF" />
-                <Link to={`/user-profile/${articles[0].author}`}><small className="profNameFeed pNFRF">{ articles[0].author }</small></Link>
```
Comment (niyoceles) [inline src/components/Articles/Feed.jsx:94]: Kindly don't remove this line, you can let this `user-profile` to separate the router of the user profile and other users.

**Review by niyoceles — DISMISSED** [review]

File: src/components/Articles/Feed.jsx [hunk 2]
```diff
@@ -70,7 +75,27 @@ export class Feed extends Component {
     return false;
   }
 
+  handleClick = (slug) => {
+    const {
+      bookmarkArticle,
+    } = this.props;
+    // const { feed } = this.props.article;
```
Comment (kodek-sleuth) [inline src/components/Articles/Feed.jsx]: This convention of commenting out code was not accepted by the team, either remove those lines or actually use them

**Review by kodek-sleuth — CHANGES_REQUESTED** [review]

**Review by kodek-sleuth — DISMISSED** [review]
I have still failed to bookmark articles on the homepage

**Review by elemanhillary-zz — DISMISSED** [review]
kindly remove the alert boxes as @dmithamo instructed and its abit confusing to they eyes of a user when he/she bookmarked or unbookmarked an article

**Review by kodek-sleuth — DISMISSED** [review]
Since this branch still has un commited changes, I failed to test it

**Pull request merged**